Atomic/util.py

```diff
@@ -842,21 +842,31 @@ def wrapper(*args, **kwargs):
 
 
 class InstallData(object):
-    if not os.path.exists(ATOMIC_INSTALL_JSON):
-        open(ATOMIC_INSTALL_JSON, 'a').close()
-
-    install_file_handle = open(ATOMIC_INSTALL_JSON, 'r')
 
     @staticmethod
-    def _read_install_data(file_handle):
+    def get_install_file_handle():
```

Comment:
Agree with this, except I think for backcompat with the previous "empty file" model we should make it something like:

```
try:
  with open(ATOMIC_INSTALL_JSON, 'r') as f:
    if os.fstat(f.fileno).st_size == 0:
      return {}
    return json.load(f)
except IOError as e:
  if e.errno == errno.ENOENT:
    return {}
  raise e